### PR TITLE
[feat] added badge

### PR DESCRIPTION
**Pull request by omjadhav85**
image

### CODE REVIEW

**Comment by omjadhav85** [comment]
Added new badge component. It has two variants:
1. For statuses - normal/online/offline/busy
2. For icon notifications with text

Feedback appreciated....thanks!!!

**Review by raghav2110 — COMMENTED** [review]
Overall code looks good.

File: styles/components/badge.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  bottom: 0.8rem;
+  right: 0.6rem;
+  top: auto;
+  left: auto;
+  height: 0.1rem;
+  width: 0.1rem;
+  border: 3px solid white;
+}
+
+.badge-status.badge-online {
```
Comment (prachi-sahani) [inline styles/components/badge.css]: I suppose ".badge-online" would have been sufficient

**Review by prachi-sahani — COMMENTED** [review]
Overall the code looks fine
Suggestion - IMO, if you move the badge little towards right and bottom(half on avatar and half outside), it would look better

File: styles/components/badge.css (hunk 1 above)
Comment (omjadhav85) [reply styles/components/badge.css]: Hey! thanks for the review. I have made the suggested change. For the position of badge, i tried to lower its position but then it does not look good with larger avatars. So made a trade-off between those two positions.

**Comment by omjadhav85** [comment]
> Overall code looks good.

**Pull request closed without merging**